elephant_vending_machine/__init__.py

```diff
@@ -4,8 +4,13 @@
 module can import it safely and the __name__ variable will always
 resolve to the correct package.
 """
+import os
 from flask import Flask
+
+IMAGE_UPLOAD_FOLDER = os.path.dirname(os.path.abspath(__file__)) + '/static/img'
```

Comment:
I like this idea, we should probably be doing something similar with the logs dir. Commenting to spur discussion on this rather than suggest a change. We may want to add a section to our Sphinx docs in the near future about changing these config values via environment variables in case a user wants to override it.

Reply:
Can you clarify on the environment variables part? Do environment variables let you override config values?

Reply:
Based on how I interpreted the [flask config docs](https://flask.palletsprojects.com/en/1.1.x/config/) it seems like they do. I believe in other platforms such as Express, environment variables will override any explicitly set config values.

Reply:
Gotcha. I thought that was the case but couldn't find the explicit documentation (clearly didn't look hard enough lol)